blablas/ep9/index.md

```diff
@@ -3,13 +3,67 @@ date: 2018-06-09
 time: 20h
 duration: "2:00:40"
 title: "UX, UI & design principles."
-tags: ["UX","UI"]
+tags: ["UX", "UI"]
 isNext: false
 video: "2151207101778310"
 published: true
 podcastUrl:
 ---
 
-[//]: # "Check this github issue on How to add Episode Notes  https://github.com/DevC-Casa/geeksblabla.com/issues/23 "
+In this episode of GeeksBlabla, Geeksblabla team, with Amine MAAROUF, Maroua EZOUHRI and EL AZIZI's Brothers, talk about The UX, UI definitions, the process to follow, tools to use,...and how to initiate the domain.
 
-...
+## Guests
+
+- Amine MAAROUF
+- Maroua EZOUHRI
+- Smail EL AZIZI
+- Youssef EL AZIZI
+
+## Notes
+
```

Comment:
for the time format, no need for seconds, you need to use the following format `h:mm`